src/Particular.ViewModelComposition.Mvc/Particular.ViewModelComposition.Mvc.csproj

```diff
@@ -4,16 +4,16 @@
     <TargetFramework>netcoreapp1.1</TargetFramework>
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>..\NServiceBus.snk</AssemblyOriginatorKeyFile>
-    <Description>TODO Add real package description</Description>
-    <!--<PackageTags>add package tags</PackageTags>-->
+    <Description>Package for view model composition in ASP.NET Core</Description>
+    <!--<PackageTags>aspnetcore ui composition viewmodel</PackageTags>-->
```

Comment:
Need to uncomment this if you want them to apply to the package.

Reply:
Uncommented